docs/functions.rst

```diff
@@ -200,9 +200,9 @@ at the beginning of the wrapped code.
 
 C helper functions
 
-``ShroudStrCopy(char *a, int la, const char *s)``
-    Copy *s* into *a*, blank fill to *la* characters
-    Truncate if *a* is too short.
+``ShroudStrCopy(char *dest, int la, const char *src, int ls)``
+    Copy *src* into *dest*, blank fill to *la* characters
```

Comment:
Two suggestions:
* La -> la
* since ``char *a`` was changed to ``char *dest``, perhaps it should be ``ld``?

Reply:
I renamed the arguments.
la -> ndest
ls -> nsrc